internal/service/atlasuser/data_source_atlas_users_test.go

```diff
@@ -20,18 +20,30 @@ func TestAccConfigDSAtlasUsers_ByOrgID(t *testing.T) {
 	var (
 		dataSourceName = "data.mongodbatlas_atlas_users.test"
 		orgID          = os.Getenv("MONGODB_ATLAS_ORG_ID")
-		users          = fetchOrgUsers(t, orgID)
 	)
-	checks := []resource.TestCheckFunc{testAccCheckMongoDBAtlasOrgWithUsersExists(dataSourceName)} // check that org has at least one user
-	checks = append(checks, dataSourceChecksForUsers(dataSourceName, orgID, users)...)
-
 	resource.Test(t, resource.TestCase{ // does not run in parallel to avoid changes in fetched users during execution
 		PreCheck:                 func() { acc.PreCheckBasic(t) },
 		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
 		Steps: []resource.TestStep{
 			{
 				Config: testAccDSMongoDBAtlasUsersByOrgID(orgID),
-				Check:  resource.ComposeAggregateTestCheckFunc(checks...),
+				ConfigStateChecks: []statecheck.StateCheck{
+					statecheck.ExpectKnownValue(
+						dataSourceName,
+						tfjsonpath.New("org_id"),
+						knownvalue.StringExact(orgID),
+					),
+					statecheck.ExpectKnownValue(
+						dataSourceName,
+						tfjsonpath.New("total_count"),
+						knownvalue.Int64Func(func(v int64) error {
+							if v < 1 {
+								return fmt.Errorf("expected total_count to be > 0, got %d", v)
+							}
+							return nil
```

Comment:
```suggestion
							if v >  0 {
								return nil
							}
							return fmt.Errorf("expected total_count to be > 0, got %d", v)
```

nit: to have code follow the error text

Reply:
Thank you! Addressed in 7fcc6fea84f6f008f65a22476f2b97c63dc19078